### ISSUE TITLE
Missing lot of databases

### ISSUE THREAD

**Issue by ala98412** [gbouras13/pharokka#313]
* pharokka version: 1.2.0
* Python version: Python 3.8.18
* Operating System: Ubuntu 18.04.6

Hi, 

I installed pharokka by the command 
```
conda install mamba
mamba install -c bioconda pharokka
```

And I also download databases through the following url
[https://zenodo.org/record/8276347/](https://zenodo.org/record/8276347/)

After doing all installations, I start to annotate phage genome by the following command
`pharokka.py -i ./p3373.genome.fasta -o ./p3373/ -d ../Pharokka_db/pharokka_v1.4.0_databases -p p3 -f -t 20`

and I got the error messages

> Starting pharokka v1.2.0
> Checking database installation.
> VFDB Databases are missing.
> INPHARED Mash Annotation File is missing.
> INPHARED Mash Sketch File is missing.
> 
> The database directory was unsuccessfully checked. Please run install_databases.

In addition, when I try to download databases by running run install_databases.py,
The command is
` install_databases.py -o database `
And I got other error messages, which is the following messages

> PHROGs Databases are missing.
> VFDB Databases are missing.
> CARD Databases are missing.
> PHROGs Annotation File is missing.
> INPHARED Mash Annotation File is missing.
> INPHARED Mash Sketch File is missing.
> Some Databases are missing.
> Downloading Pharokka Database
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
> 100   295  100   295    0     0    331      0 --:--:-- --:--:-- --:--:--   331
> 
> gzip: stdin: not in gzip format
> tar: Child returned status 1
> tar: Error is not recoverable: exiting now

I am pretty sure that I have every required VFDB files
```
-rw-r--r-- 1 JuiHung JuiHung   13353794  八  21 13:35 vfdb
-rw-r--r-- 1 JuiHung JuiHung          4  八  21 13:35 vfdb.dbtype
-rw-r--r-- 1 JuiHung JuiHung    4621196  八  21 13:35 vfdb_h
-rw-r--r-- 1 JuiHung JuiHung          4  八  21 13:35 vfdb_h.dbtype
-rw-r--r-- 1 JuiHung JuiHung     485796  八  21 13:35 vfdb_h.index
-rw-r--r-- 1 JuiHung JuiHung     498782  八  21 13:35 vfdb.index
-rw-r--r-- 1 JuiHung JuiHung     962267  八  21 13:35 vfdb.lookup
-rw-r--r-- 1 JuiHung JuiHung    5281474  八  21 13:33 VFDB_setB_pro.fas.gz
-rw-r--r-- 1 JuiHung JuiHung         20  八  21 13:35 vfdb.source
```

Maybe I miss some part I guess.
Please help me to solve the problem.
Many thanls.

Best, 
Jui-Hung

**Comment by gbouras13** [comment]
Hi Jui-Hung,

It looks like you are using quite an old version (V1.2.0) of Pharokka with newer database versions.

I'd recommend trying to install the latest version as follows:

```
mamba install -c bioconda pharokka==1.5.1
```

and hopefully it will to download when you run

```
install_databases.py -o pharokka_db
```

George

**Comment by ala98412** [comment]
Hi George,

I am trying to install pharokka 1.5, but there is too much tool conflicts.


```
Looking for: ['pharokka==1.5']

pkgs/main/linux-64                                            No change
pkgs/main/noarch                                              No change
pkgs/r/linux-64                                               No change
pkgs/r/noarch                                                 No change
bioconda/linux-64                                             No change
bioconda/noarch                                      5.0MB @   5.1MB/s  1.1s

Pinned packages:
  - python 3.8.*


Could not solve for environment specs
The following packages are incompatible
├─ mamba is installable and it requires
│  ├─ libmambapy 1.5.5 py38h5cd715c_0, which requires
│  │  └─ libmamba 1.5.5 had39da4_0, which requires
│  │     └─ libcurl >=8.5.0,<9.0a0 , which can be installed;
│  └─ openssl >=3.2.0,<4.0a0 , which can be installed;
└─ pharokka 1.5  is not installable because it requires
   └─ dnaapler >=0.3.2 , which requires
      └─ blast >=2.10  but there are no viable options
         ├─ blast 2.10.1 would require
         │  └─ curl >=7.71.1,<8.0a0  but there are no viable options
         │     ├─ curl [7.71.0|7.71.1] would require
         │     │  └─ libcurl [7.71.0 h20c2e04_0|7.71.1 h20c2e04_1], which requires
         │     │     └─ openssl >=1.1.1g,<1.1.2a , which conflicts with any installable versions previously reported;
         │     ├─ curl [7.71.1|7.78.0] would require
         │     │  └─ libcurl [7.71.1 h303737a_2|7.78.0 h0b77cf5_0], which requires
         │     │     └─ openssl >=1.1.1k,<1.1.2a , which conflicts with any installable versions previously reported;
         │     ├─ curl 7.80.0 would require
         │     │  └─ libcurl 7.80.0 h0b77cf5_0, which requires
         │     │     └─ openssl >=1.1.1l,<1.1.2a , which conflicts with any installable versions previously reported;
         │     ├─ curl 7.82.0 would require
         │     │  └─ libcurl 7.82.0 h0b77cf5_0, which requires
         │     │     └─ openssl >=1.1.1n,<1.1.2a , which conflicts with any installable versions previously reported;
         │     ├─ curl 7.83.0 would require
         │     │  └─ libcurl 7.83.0 h0b77cf5_0, which does not exist (perhaps a missing channel);
         │     ├─ curl [7.84.0|7.85.0] would require
         │     │  └─ libcurl [7.84.0 h91b91d3_0|7.85.0 h91b91d3_0], which requires
         │     │     └─ openssl >=1.1.1q,<1.1.2a , which conflicts with any installable versions previously reported;
         │     ├─ curl [7.86.0|7.87.0] would require
         │     │  └─ libcurl [7.86.0 h91b91d3_0|7.87.0 h91b91d3_0], which requires
         │     │     └─ openssl >=1.1.1s,<1.1.2a , which conflicts with any installable versions previously reported;
         │     ├─ curl [7.88.1|8.1.1] would require
         │     │  └─ libcurl [7.88.1 h91b91d3_2|8.1.1 h91b91d3_1|8.1.1 h91b91d3_2], which requires
         │     │     └─ openssl >=1.1.1u,<1.1.2a , which conflicts with any installable versions previously reported;
         │     ├─ curl [7.88.1|8.1.1] would require
         │     │  └─ libcurl [7.88.1 h91b91d3_0|8.1.1 h91b91d3_0], which requires
         │     │     └─ openssl >=1.1.1t,<1.1.2a , which conflicts with any installable versions previously reported;
         │     ├─ curl 7.88.1 would require
         │     │  └─ libcurl 7.88.1 h251f7ec_1, which conflicts with any installable versions previously reported;
         │     └─ curl 7.88.1 would require
         │        └─ libcurl 7.88.1 h251f7ec_2, which conflicts with any installable versions previously reported;
         ├─ blast [2.11.0|2.12.0] would require
         │  └─ curl [>=7.75.0,<8.0a0 |>=7.77.0,<8.0a0 ], which cannot be installed (as previously explained);
         ├─ blast 2.12.0 would require
         │  └─ curl >=7.81.0,<8.0a0 , which cannot be installed (as previously explained);
         ├─ blast 2.13.0 would require
         │  └─ curl >=7.83.1,<8.0a0 , which cannot be installed (as previously explained);
         ├─ blast [2.13.0|2.14.0] would require
         │  └─ curl >=7.88.1,<8.0a0 , which cannot be installed (as previously explained);
         └─ blast [2.14.0|2.14.1|2.15.0] would require
            └─ curl but there are no viable options
               ├─ curl 7.55.1 would require
               │  └─ openssl >=1.0.2m,<1.0.3a , which conflicts with any installable versions previously reported;
               ├─ curl 7.55.1 would require
               │  └─ openssl 1.0.* , which conflicts with any installable versions previously reported;
               ├─ curl [7.57.0|7.58.0] would require
               │  └─ libcurl [7.57.0 h1ad7b7a_0|7.58.0 h1ad7b7a_0], which requires
               │     └─ openssl >=1.0.2n,<1.0.3a , which conflicts with any installable versions previously reported;
               ├─ curl [7.59.0|7.60.0|7.61.0] would require
               │  └─ libcurl [7.59.0 h1ad7b7a_0|7.60.0 h1ad7b7a_0|7.61.0 h1ad7b7a_0], which requires
               │     └─ openssl >=1.0.2o,<1.0.3a , which conflicts with any installable versions previously reported;
               ├─ curl [7.61.1|7.62.0|7.63.0] would require
               │  └─ libcurl [7.61.1 h20c2e04_0|7.62.0 h20c2e04_0|7.63.0 h20c2e04_0|7.63.0 h20c2e04_1000], which requires
               │     └─ openssl >=1.1.1a,<1.1.2a , which conflicts with any installable versions previously reported;
               ├─ curl [7.64.0|7.64.1] would require
               │  └─ libcurl [7.64.0 h20c2e04_2|7.64.1 h20c2e04_0], which requires
               │     └─ openssl >=1.1.1b,<1.1.2a , which conflicts with any installable versions previously reported;
               ├─ curl [7.65.2|7.65.3] would require
               │  └─ libcurl [7.65.2 h20c2e04_0|7.65.3 h20c2e04_0], which requires
               │     └─ openssl >=1.1.1c,<1.1.2a , which conflicts with any installable versions previously reported;
               ├─ curl [7.67.0|7.68.0|7.69.1] would require
               │  └─ libcurl [7.67.0 h20c2e04_0|7.68.0 h20c2e04_0|7.69.1 h20c2e04_0], which requires
               │     └─ openssl >=1.1.1d,<1.1.2a , which conflicts with any installable versions previously reported;
               ├─ curl [7.71.0|7.71.1], which cannot be installed (as previously explained);
               ├─ curl [7.71.1|7.78.0], which cannot be installed (as previously explained);
               ├─ curl 7.80.0, which cannot be installed (as previously explained);
               ├─ curl 7.82.0, which cannot be installed (as previously explained);
               ├─ curl 7.83.0, which cannot be installed (as previously explained);
               ├─ curl [7.84.0|7.85.0], which cannot be installed (as previously explained);
               ├─ curl [7.86.0|7.87.0], which cannot be installed (as previously explained);
               ├─ curl [7.88.1|8.1.1], which cannot be installed (as previously explained);
               ├─ curl [7.88.1|8.1.1], which cannot be installed (as previously explained);
               ├─ curl 7.88.1, which cannot be installed (as previously explained);
               ├─ curl 7.88.1, which cannot be installed (as previously explained);
               ├─ curl 8.1.1 would require
               │  └─ libcurl 8.1.1 h251f7ec_2, which conflicts with any installable versions previously reported;
               ├─ curl 8.2.1 would require
               │  └─ libcurl 8.2.1 h91b91d3_0, which requires
               │     └─ openssl >=1.1.1v,<1.1.2a , which conflicts with any installable versions previously reported;
               ├─ curl 8.2.1 would require
               │  └─ libcurl 8.2.1 h251f7ec_0, which conflicts with any installable versions previously reported;
               ├─ curl 8.4.0 would require
               │  └─ libcurl 8.4.0 h251f7ec_0, which conflicts with any installable versions previously reported;
               └─ curl 8.4.0 would require
                  └─ libcurl 8.4.0 h251f7ec_1, which conflicts with any installable versions previously reported.

```


I install dnaapler through the following command,
`conda install -c bioconda dnaapler`
but the error message still appear.

Is there any easier way to install pharokka?

Thank you

**Comment by gbouras13** [comment]
Hi @ala98412 ,

Usually, if you are getting this many conflicts, a better approach is just to re-install mamba completely usually. 

George

**Comment by jwm6** [comment]
Hey gang, 

I have been running into this same issue. No matter how many times I install or reinstall or uninstall and reinstall, whatever, I get the same errors.  I believe pharokka is downloaded into one working directory, but the databases are stored elsewhere, so when the program goes to run the databases, it cant find them because somehow got stored somewhere else.  I can tell that I have downloaded the same databases in like 3 different working directories to just hedge my bets at this moment, but alas, no dice . Please help troubleshoot.

(base) jwyhaymar@JWWMH:~/miniconda3/bin$ install_databases.py -o /home/jwyhaymar/Pharokka_env
2024-01-17 16:30:55.049 | INFO     | databases:instantiate_install:111 - Checking Pharokka database installation in /home/jwyhaymar/Pharokka_env.
2024-01-17 16:30:55.049 | INFO     | databases:check_db_installation:213 - PHROGs Databases are missing.
2024-01-17 16:30:55.050 | INFO     | databases:check_db_installation:220 - VFDB Databases are missing.
2024-01-17 16:30:55.050 | INFO     | databases:check_db_installation:227 - CARD Databases are missing.
2024-01-17 16:30:55.050 | INFO     | databases:check_db_installation:233 - PHROGs Annotation File is missing.
2024-01-17 16:30:55.050 | INFO     | databases:check_db_installation:239 - INPHARED Mash Annotation File is missing.
2024-01-17 16:30:55.051 | INFO     | databases:check_db_installation:245 - INPHARED Mash Sketch File is missing.
2024-01-17 16:30:55.051 | INFO     | databases:instantiate_install:116 - Some Databases are missing.
2024-01-17 16:30:55.051 | INFO     | databases:instantiate_install:121 - Downloading Pharokka Databases from https://zenodo.org/record/8276347/files/pharokka_v1.4.0_databases.tar.gz.
|████████████████████████████████████████| 571.7M/571.7M [100%] in 52.7s (10.84M/s)
2024-01-17 16:31:49.632 | INFO     | databases:instantiate_install:132 - Database file download OK: c21144209b993c06fae2dac906d73b96
2024-01-17 16:31:49.633 | INFO     | databases:instantiate_install:138 - Extracting DB tarball: file=/home/jwyhaymar/Pharokka_env/pharokka_v1.4.0_databases.tar.gz, output=/home/jwyhaymar/Pharokka_env
(base) jwyhaymar@JWWMH:~/miniconda3/bin$ pharokka.py -i Bas26_Parent.fasta -o  -t 12
usage: pharokka.py [-h] [-i INFILE] [-o OUTDIR] [-d DATABASE] [-t THREADS] [-f] [-p PREFIX]
                   [-l LOCUSTAG] [-g GENE_PREDICTOR] [-m] [-s] [-c CODING_TABLE]
                   [-e EVALUE] [--fast] [--mmseqs2_only] [--meta_hmm] [--dnaapler]
                   [--custom_hmm CUSTOM_HMM] [--genbank] [--terminase]
                   [--terminase_strand TERMINASE_STRAND]
                   [--terminase_start TERMINASE_START] [--skip_extra_annotations]
                   [--skip_mash] [--minced_args MINCED_ARGS]
                   [--mash_distance MASH_DISTANCE] [-V] [--citation]
pharokka.py: error: argument -o/--outdir: expected one argument
(base) jwyhaymar@JWWMH:~/miniconda3/bin$ pharokka.py -i Bas26_Parent.fasta -o phar_out -t 12
2024-01-17 16:34:32.999 | INFO     | __main__:main:84 - Starting Pharokka v1.6.0
2024-01-17 16:34:33.000 | INFO     | __main__:main:85 - Command executed: Namespace(infile='Bas26_Parent.fasta', outdir='phar_out', database='Default', threads='12', force=False, prefix='Default', locustag='Default', gene_predictor='phanotate', meta=False, split=False, coding_table='11', evalue='1E-05', fast=False, mmseqs2_only=False, meta_hmm=False, dnaapler=False, custom_hmm='', genbank=False, terminase=False, terminase_strand='nothing', terminase_start='nothing', skip_extra_annotations=False, skip_mash=False, minced_args='', mash_distance=0.2, citation=False)
2024-01-17 16:34:33.000 | INFO     | __main__:main:86 - Repository homepage is https://github.com/gbouras13/pharokka
2024-01-17 16:34:33.000 | INFO     | __main__:main:87 - Written by George Bouras: george.bouras@adelaide.edu.au
2024-01-17 16:34:33.001 | INFO     | __main__:main:89 - Checking database installation in /home/jwyhaymar/miniconda3/bin/pharokka.py/../databases/.
2024-01-17 16:34:33.001 | INFO     | databases:check_db_installation:213 - PHROGs Databases are missing.
2024-01-17 16:34:33.001 | INFO     | databases:check_db_installation:220 - VFDB Databases are missing.
2024-01-17 16:34:33.002 | INFO     | databases:check_db_installation:227 - CARD Databases are missing.
2024-01-17 16:34:33.002 | INFO     | databases:check_db_installation:233 - PHROGs Annotation File is missing.
2024-01-17 16:34:33.002 | INFO     | databases:check_db_installation:239 - INPHARED Mash Annotation File is missing.
2024-01-17 16:34:33.002 | INFO     | databases:check_db_installation:245 - INPHARED Mash Sketch File is missing.
2024-01-17 16:34:33.003 | ERROR    | __main__:main:94 - The database directory was unsuccessfully checked. Please run install_databases.py.
(base) jwyhaymar@JWWMH:~/miniconda3/bin$ pharokka.py -i Bas26_Parent.fasta -o phar_out -t 12
2024-01-17 16:35:24.269 | ERROR    | input_commands:instantiate_dirs:196 - The output directory phar_out already exists and force was not specified. Please specify -f or --force to overwrite it.
(base) jwyhaymar@JWWMH:~/miniconda3/bin$ pharokka.py -i Bas26_Parent.fasta -o phar_out -t 12
-f
2024-01-17 16:35:31.639 | INFO     | input_commands:instantiate_dirs:183 - Removing output directory phar_out as -f or --force was specified.
2024-01-17 16:35:31.644 | INFO     | __main__:main:84 - Starting Pharokka v1.6.0
2024-01-17 16:35:31.644 | INFO     | __main__:main:85 - Command executed: Namespace(infile='Bas26_Parent.fasta', outdir='phar_out', database='Default', threads='12', force=True, prefix='Default', locustag='Default', gene_predictor='phanotate', meta=False, split=False, coding_table='11', evalue='1E-05', fast=False, mmseqs2_only=False, meta_hmm=False, dnaapler=False, custom_hmm='', genbank=False, terminase=False, terminase_strand='nothing', terminase_start='nothing', skip_extra_annotations=False, skip_mash=False, minced_args='', mash_distance=0.2, citation=False)
2024-01-17 16:35:31.644 | INFO     | __main__:main:86 - Repository homepage is https://github.com/gbouras13/pharokka
2024-01-17 16:35:31.645 | INFO     | __main__:main:87 - Written by George Bouras: george.bouras@adelaide.edu.au
2024-01-17 16:35:31.645 | INFO     | __main__:main:89 - Checking database installation in /home/jwyhaymar/miniconda3/bin/pharokka.py/../databases/.
2024-01-17 16:35:31.645 | INFO     | databases:check_db_installation:213 - PHROGs Databases are missing.
2024-01-17 16:35:31.646 | INFO     | databases:check_db_installation:220 - VFDB Databases are missing.
2024-01-17 16:35:31.646 | INFO     | databases:check_db_installation:227 - CARD Databases are missing.
2024-01-17 16:35:31.646 | INFO     | databases:check_db_installation:233 - PHROGs Annotation File is missing.
2024-01-17 16:35:31.646 | INFO     | databases:check_db_installation:239 - INPHARED Mash Annotation File is missing.
2024-01-17 16:35:31.647 | INFO     | databases:check_db_installation:245 - INPHARED Mash Sketch File is missing.
2024-01-17 16:35:31.647 | ERROR    | __main__:main:94 - The database directory was unsuccessfully checked. Please run install_databases.py.
(base) jwyhaymar@JWWMH:~/miniconda3/bin$ install_databases.py -d
2024-01-17 16:35:41.551 | INFO     | databases:instantiate_install:111 - Checking Pharokka database installation in /home/jwyhaymar/miniconda3/bin/../databases/.
2024-01-17 16:35:41.551 | INFO     | databases:instantiate_install:114 - All Pharokka Databases have already been Downloaded and Checked.
(base) jwyhaymar@JWWMH:~/miniconda3/bin$ pharokka.py -i Bas26_Parent.fasta -o phar_out -d hom
e/jwyhaymar/miniconda3/bin/Pharokka_env -t 8
2024-01-17 16:38:04.577 | ERROR    | input_commands:instantiate_dirs:196 - The output directory phar_out already exists and force was not specified. Please specify -f or --force to overwrite it.
(base) jwyhaymar@JWWMH:~/miniconda3/bin$ pharokka.py -i Bas26_Parent.fasta -o phar_out -d home/jwyhaymar/miniconda3/bin/Pharokka_env -t 8 -f
2024-01-17 16:38:10.670 | INFO     | input_commands:instantiate_dirs:183 - Removing output directory phar_out as -f or --force was specified.
2024-01-17 16:38:10.674 | INFO     | __main__:main:84 - Starting Pharokka v1.6.0
2024-01-17 16:38:10.675 | INFO     | __main__:main:85 - Command executed: Namespace(infile='Bas26_Parent.fasta', outdir='phar_out', database='home/jwyhaymar/miniconda3/bin/Pharokka_env', threads='8', force=True, prefix='Default', locustag='Default', gene_predictor='phanotate', meta=False, split=False, coding_table='11', evalue='1E-05', fast=False, mmseqs2_only=False, meta_hmm=False, dnaapler=False, custom_hmm='', genbank=False, terminase=False, terminase_strand='nothing', terminase_start='nothing', skip_extra_annotations=False, skip_mash=False, minced_args='', mash_distance=0.2, citation=False)
2024-01-17 16:38:10.675 | INFO     | __main__:main:86 - Repository homepage is https://github.com/gbouras13/pharokka
2024-01-17 16:38:10.675 | INFO     | __main__:main:87 - Written by George Bouras: george.bouras@adelaide.edu.au
2024-01-17 16:38:10.675 | INFO     | __main__:main:89 - Checking database installation in home/jwyhaymar/miniconda3/bin/Pharokka_env.
2024-01-17 16:38:10.676 | INFO     | databases:check_db_installation:213 - PHROGs Databases are missing.
2024-01-17 16:38:10.676 | INFO     | databases:check_db_installation:220 - VFDB Databases are missing.
2024-01-17 16:38:10.676 | INFO     | databases:check_db_installation:227 - CARD Databases are missing.
2024-01-17 16:38:10.677 | INFO     | databases:check_db_installation:233 - PHROGs Annotation File is missing.
2024-01-17 16:38:10.677 | INFO     | databases:check_db_installation:239 - INPHARED Mash Annotation File is missing.
2024-01-17 16:38:10.677 | INFO     | databases:check_db_installation:245 - INPHARED Mash Sketch File is missing.
2024-01-17 16:38:10.677 | ERROR    | __main__:main:94 - The database directory was unsuccessfully checked. Please run install_databases.py.
(base) jwyhaymar@JWWMH:~/miniconda3/bin$ pharokka.py -i Bas26_Parent.fasta -o phar_out -d /ho
me/jwyhaymar/miniconda3/bin/Pharokka_env -t 8 -f
2024-01-17 16:38:36.233 | INFO     | input_commands:instantiate_dirs:183 - Removing output directory phar_out as -f or --force was specified.
2024-01-17 16:38:36.237 | INFO     | __main__:main:84 - Starting Pharokka v1.6.0
2024-01-17 16:38:36.238 | INFO     | __main__:main:85 - Command executed: Namespace(infile='Bas26_Parent.fasta', outdir='phar_out', database='/home/jwyhaymar/miniconda3/bin/Pharokka_env', threads='8', force=True, prefix='Default', locustag='Default', gene_predictor='phanotate', meta=False, split=False, coding_table='11', evalue='1E-05', fast=False, mmseqs2_only=False, meta_hmm=False, dnaapler=False, custom_hmm='', genbank=False, terminase=False, terminase_strand='nothing', terminase_start='nothing', skip_extra_annotations=False, skip_mash=False, minced_args='', mash_distance=0.2, citation=False)
2024-01-17 16:38:36.238 | INFO     | __main__:main:86 - Repository homepage is https://github.com/gbouras13/pharokka
2024-01-17 16:38:36.238 | INFO     | __main__:main:87 - Written by George Bouras: george.bouras@adelaide.edu.au
2024-01-17 16:38:36.238 | INFO     | __main__:main:89 - Checking database installation in /home/jwyhaymar/miniconda3/bin/Pharokka_env.
2024-01-17 16:38:36.239 | INFO     | databases:check_db_installation:213 - PHROGs Databases are missing.
2024-01-17 16:38:36.239 | INFO     | databases:check_db_installation:220 - VFDB Databases are missing.
2024-01-17 16:38:36.239 | INFO     | databases:check_db_installation:227 - CARD Databases are missing.
2024-01-17 16:38:36.239 | INFO     | databases:check_db_installation:233 - PHROGs Annotation File is missing.
2024-01-17 16:38:36.240 | INFO     | databases:check_db_installation:239 - INPHARED Mash Annotation File is missing.
2024-01-17 16:38:36.240 | INFO     | databases:check_db_installation:245 - INPHARED Mash Sketch File is missing.
2024-01-17 16:38:36.240 | ERROR    | __main__:main:94 - The database directory was unsuccessfully checked. Please run install_databases.py.
(base) jwyhaymar@JWWMH:~/miniconda3/bin$ install_databases.py
2024-01-17 16:38:47.290 | INFO     | __main__:<module>:40 - --outdir was not specified.
2024-01-17 16:38:47.290 | INFO     | __main__:<module>:41 - Downloading databases to the default directory anyway.
2024-01-17 16:38:47.290 | INFO     | databases:instantiate_install:111 - Checking Pharokka database installation in /home/jwyhaymar/miniconda3/bin/../databases/.
2024-01-17 16:38:47.291 | INFO     | databases:instantiate_install:114 - All Pharokka Databases have already been Downloaded and Checked.

**Comment by gbouras13** [comment]
Maybe try `install_databases.py -o /home/jwyhaymar/Pharokka_env_dbs`

Then use `-d  /home/jwyhaymar/Pharokka_env_dbs` with `pharokka.py`?

It's a bit hard to follow with the crossed out text above so I'm not sure if you have tried that.

George

**Comment by gbouras13** [comment]
The other thing is to look at what is actually in `/ho
me/jwyhaymar/miniconda3/bin/Pharokka_env`  with `ls` - the download might have failed. Then I'd try a manual download from zenodo as outlined in the documentation.

George

**Comment by gbouras13** [comment]
The only other problem I can think of is if you are  storing the database/env on a different partition to the program itself, but doesn't look like it.

George

**Comment by jwm6** [comment]
Hey george and gang, 
I got it to work.  Sadly, I believe I downloaded the databases in multiple directories, just hedging my bet (i am a toddler with this stuff and doubt i will be able to figure out where the multiples are), but I downloaded the databases (both ways below) without conda activated, then activated conda to run pharokka like you have in the read me:

install_databases.py -d

and 

wget "https://zenodo.org/record/8267900/files/pharokka_v1.4.0_databases.ta
r.gz"
tar -xzf pharokka_v1.4.0_databases.tar.gz
